source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -42,10 +42,9 @@ content: |
         .. code-block:: python
            :emphasize-lines: 4
 
-           from bson import binary
-
            path = "./master-key.txt"
-           local_master_key = binary.Binary(open(path, "rb").read(96))
+           with open(path, "rb") as f:
+             local_master_key = f.read()
```

Comment:
Using a with-statement is more idiomatic.

Reply:
thanks for pointing this out; resolving this thread